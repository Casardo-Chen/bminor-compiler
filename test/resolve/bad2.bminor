// multiple definition of function

helper: function integer ()= {
    return 42;
}

helper: function integer ()= {
    return 24;
}