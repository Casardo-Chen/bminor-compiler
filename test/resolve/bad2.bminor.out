resolve error: duplicate definitions of helper is invalid.
1 resolve error(s) found.
